src/components/UploadSection.svelte

```diff
@@ -27,16 +28,29 @@
         <div class="box">
             <div class="rows is-centered">
                 <div class="row has-text-centered">
+                    <div class="columns">
+                        <p class="column">Name</p>
+                        <p class="column">Size</p>
+                        <p class="column">Status</p>
+                    </div>
                     {#if uploadedFiles}
-                        {#each uploadedFiles as { ObjectKey }}
-                            <p>{ObjectKey}</p>
+                        {#each uploadedFiles as { ObjectKey, StatusName }}
+                        <div class="columns">
+                            <p class="column">{ObjectKey}</p>
+                            <p class="column">-</p>
+                            <p class="column">{StatusName}</p>
+                        </div>
```

Comment:
Bulma has a [Table element](https://bulma.io/documentation/elements/table/). But it is ok also

Reply:
Also if you don't have some data (e.g. size) in request, feel free to say it. I see that we would need to add some fileds to a table in a db.
e.g. `size_bytes` + metadata?